pkg/exporter/metrics/monthly_active_users.go

```diff
@@ -11,23 +11,31 @@ const (
 	tenantTotalMonthlyActiveUsers = "tenant_total_monthly_active_users"
 )
 
-func monthlyActiveUsersCounterMetric(namespace, subsystem string, applications []*management.Client) *prometheus.Counter {
-	m := prometheus.NewCounter(
-		prometheus.CounterOpts{
+func monthlyActiveUsersGaugeMetric(namespace, subsystem string, applications []*management.Client) *prometheus.Gauge {
+	m := prometheus.NewGauge(
+		prometheus.GaugeOpts{
 			Name: prometheus.BuildFQName(namespace, subsystem, tenantTotalMonthlyActiveUsers),
 			Help: "The total number of monthly active users on the tenant.",
 		})
 	return &m
 }
 
 func processMonthlyActiveUsers(m *Metrics, users []*management.User) error {
+	if len(users) == 0 {
```

Comment:
Question: is this check needed? Wouldn't we want the metric to go to 0 since it's a Gauge now?

Reply:
@tfadeyi it should be fixed in e7949a4 :)